defaults/main.yml

```diff
@@ -5,6 +5,7 @@ rhc_baseurl: null
 rhc_environments: []
 rhc_insights:
   autoupdate: true
+  ansible_host: null
```

Comment:
since there are still changes to do, would you please move this `ansible_host` line one line up, so the list of items in `rhc_insights` is still alphasorted? :) (yes, nitpicks)

Reply:
More nitpicks, please.